parity-clib-example/main.cpp

```diff
@@ -8,8 +11,8 @@ int main() {
     ParityParams cfg = { 0 };
     cfg.on_client_restart_cb = on_restart;
 
-    const char* args[] = {"--light"};
-    size_t str_lens[] = {7};
+    const char* args[] = {"--no-ipc"};
```

Comment:
I removed `--light` because `parity_versionInfo` isn't available for the light client (don't really know why).
The `--no-ipc` is here just to give an example usage for the arguments.